### DISCUSSION TITLE
Flatpak version of jellyfin-vue

### DISCUSSION THREAD

**Discussion by exil0867** [jellyfin/jellyfin-vue#1923] [Questions]
Hello!

I am a user of the [Jellyfin Media Player Flatpak](https://github.com/flathub/com.github.iwalton3.jellyfin-media-player), and I have been following the development of the jellyfin-vue. I understand that it's still in an early stage, But yesterday after noticing the merge of Vue 3 upgrade PR i've decided to self-host it, try it out and i ended up really liking it, it made me consider switching to it for the time being.

I wanted to inquire if there are any plans to create a Flatpak version for jellyfin-vue, (Preferably hosted on the Flathub beta remote branch). This would make it more accessible for users who are interested in trying the new front-end and providing valuable feedback.

Thanks.

**Comment by ThibaultNocchi** [comment]
For a Flatpak version, we'd need the local players like JMP or Tauri to work correctly with JF-Vue, which I'm not sure they do cause I haven't tested that part in a while.
Once that does, it should be possible.

**Reply by exil0867** [reply]
I think @iwalton3 can enlighten us about this 😁.

**Reply by iwalton3** [reply]
I published documentation that could assist with such an integration. It does require some work by the vue maintainers to integrate with the native player.

 - https://github.com/jellyfin/jellyfin-media-player/blob/master/for-web-developers.md
 - https://gist.github.com/iwalton3/8f0f4c5caa51b08975e9b61a0a0047bc
